install_config/persistent_storage/persistent_storage_aws.adoc

```diff
@@ -142,3 +142,12 @@ formatted with the given file system.
 
 This allows using unformatted AWS volumes as persistent volumes, because
 {product-title} formats them before the first use.
+
+=== Maximum Number of EBS Volumes on a Node
+By default {product-title} supports a maximum of 39 EBS volumes attached to one node. This limit is
+consistent with the link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/volume_limits.html#linux-specific-volume-limits[AWS Volume limits]
+
+{product-title} can however be configured to have a higher limit by setting the environment variable *KUBE_MAX_PD_VOLS*. AWS however
+requires a particular naming scheme(link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/device_naming.html[AWS Device naming]) for attached devices
+and that device naming scheme only supports a maximum of 52 volumes. So - theoretical upper limit of volumes attachable to a node via
```

Comment:
Change 'for attached devices and that device naming scheme only supports a maximum of 52 volumes' to 'for attached devices that only supports a maximum of 52 volumes'